src/main/java/teammates/ui/template/StudentFeedbackResultsQuestionWithResponses.java

```diff
@@ -1,5 +1,6 @@
 package teammates.ui.template;
 
+import java.util.ArrayList;
 import java.util.List;
 
 public class StudentFeedbackResultsQuestionWithResponses {
```

Comment:
If u don't mind. Can u also add a javadoc for this class. Like mentioning we separate the responses to self responses and others.

> We are begin to fix the missing javadoc now. It is also a good practice for u to dig into the code :)